src/RedisClusterOptions.php

```diff
@@ -222,4 +224,14 @@ public function setPassword(string $password): void
     {
         $this->password = $password;
     }
+
+    public function getSslContext(): array
+    {
+        return $this->sslContext;
```

Comment:
Is there any way to document this better? :thinking: 

at least `array<string, mixed>`, I suppose? Or `array{verify_peer: bool, ...array<string, mixed>}`?

Reply:
Yeah it makes sense to document the field and getter return type as array<string, mixed>.

The single source of truth for recognized options in this array would be: https://www.php.net/manual/en/context.ssl.php. Would it be useful to put this in the docstring?

Reply:
Before throwing effort at it, let's wait for @boesing, who's the main maintainer here :) 

Want to see if we should support the whole SSL context, or just parts of it.

Reply:
Oh yah, thats exactly what I requested below.
I'd say lets only return an object, that would allow us to consume that context in a more OOP way. I'd like to prevent arrays as much as possible. IMO we should allow an array shape and an object during instantiation, we could provide a proper Options array shape on class-level, I might want to require that in the next major version anyways for *every* options of an adapter. That would basically allow us to have proper type-safety via static analysis and allows us to have an object instantiated from an array.

Since `laminas-cache` depends on `laminas-stdlib`, we could even implement https://github.com/laminas/laminas-stdlib/blob/ec5f3d3f1a2c44575d3369981ff5a1058aacf77d/src/ArraySerializableInterface.php to have a method to pass the SSL context as an array to the `RedisCluster#__construct`.

WDYT?